src/css/spectrum/spectrum.css

```diff
@@ -13,7 +13,7 @@ License: MIT
     *display: inline;
     *zoom: 1;
     /* https://github.com/bgrins/spectrum/issues/40 */
-    z-index: 9999994;
+    z-index: 20000;
```

Comment:
So if necessary let's move this to the override file.

I'm not sure we need the bootstrap tooltip to be displayed on top of the color picker. Do we have bootstrap tooltip inside the color picker ?